src/sinks/tsd-query-log-sink.ts

```diff
@@ -72,29 +74,35 @@ export class TsdQueryLogSink implements tsdDef.Sink {
     private static stringify(metricsEvent:tsdDef.MetricsEvent):string {
         var transformedMetricsEvent = TsdQueryLogSink.transformMetricsEvent(metricsEvent);
```

Comment:
Ah you're saying this should really be named `TsdQueryLogStenoSink` or similar?

As an aside, I'm planning to drop steno with the second part of these changes with 2G.